src/Microsoft.DocAsCode.Build.Engine/Template.cs

```diff
@@ -235,15 +217,15 @@ private static ITemplatePreprocessor CreatePreprocessor(ResourceCollection resou
             return new TemplateJintPreprocessor(resourceCollection, scriptResource, context);
         }
 
-        private static ITemplateRenderer CreateRenderer(ResourceCollection resourceCollection, TemplateRendererResource templateResource)
+        private static ITemplateRenderer CreateRenderer(ResourceCollection resourceCollection, TemplateRendererResource templateResource, string templateName)
```

Comment:
if can, how about passing TemplateRendererResource as param name?

---
In reply to: [109278891](https://github.com/dotnet/docfx/pull/1513#discussion_r109278891) [](ancestors = 109278891)